src/v3/test/integration/identify-with-password.test.tsx

```diff
@@ -327,4 +327,23 @@ describe('identify-with-password', () => {
       );
     });
   });
+
+  it('should only send one api request when submit button is double clicked', async () => {
+    const {
+      authClient, user, findByTestId, findByText,
+    } = await setup({ mockResponse });
+
+    const usernameEl = await findByTestId('identifier') as HTMLInputElement;
+    const passwordEl = await findByTestId('credentials.passcode') as HTMLInputElement;
+    const submitButton = await findByText('Sign in', { selector: 'button' });
+
+    await user.type(usernameEl, 'testuser@okta.com');
+    expect(usernameEl.value).toEqual('testuser@okta.com');
+    await user.type(passwordEl, 'fake-password');
+    expect(passwordEl.value).toEqual('fake-password');
+    await user.click(submitButton);
+    await user.click(submitButton);
```

Comment:
I think there might be an actual double click event. If there is, we should use that